Packs/CortexXDR/Playbooks/Cortex_XDR_-_Retrieve_File_v2.yml

```diff
@@ -0,0 +1,382 @@
+    type: regular
+    task:
+      id: 7cb1327a-ad50-4f2d-8a36-27848573df25
+      version: -1
+      name: Retrieve files
+      description: Retrieves files from selected endpoints. You can retrieve up to 20 files, from no more than 10 endpoints. At least one endpoint ID and one file path are necessary in order to run the command. After running this command, you can use the xdr-action-status-get command with returned action_id, to check the action status.
+      script: '|||xdr-file-retrieve'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      endpoint_ids:
+        complex:
+          root: inputs.endpoint_ids
+          transformers:
+          - operator: uniq
+      generic_file_path:
+        complex:
+          root: inputs.file_path
+          transformers:
+          - operator: uniq
+      interval_in_seconds:
+        simple: "60"
+      linux_file_paths:
+        complex:
+          root: inputs.linux_file_paths
+          transformers:
+          - operator: uniq
+      mac_file_paths:
+        complex:
+          root: inputs.mac_file_paths
+          transformers:
+          - operator: uniq
+      timeout_in_seconds:
+        simple: "600"
+      windows_file_paths:
+        complex:
+          root: inputs.windows_file_paths
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 395
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: a2712ae7-1100-4545-85a7-43770ff6fbd7
+    type: title
+    task:
+      id: a2712ae7-1100-4545-85a7-43770ff6fbd7
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 1095
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "5":
+    id: "5"
+    taskid: 069bc12b-b96b-47ef-86dd-c000cdc518b6
+    type: condition
+    task:
+      id: 069bc12b-b96b-47ef-86dd-c000cdc518b6
+      version: -1
+      name: Retrieve file finished
+      description: Checks if the link to the file is ready.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "6"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isExists
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.RetrievedFiles.status
```

Comment:
QQ: in case of an error - this be defined as well? 
If so, maybe we should check against possible values.